src/main/java/ml/shifu/shifu/enhancer/EnhanceCallable.java

```diff
@@ -0,0 +1,12 @@
+package ml.shifu.shifu.enhancer;
+
+/**
+ * EnhanceCallable is used to define basic interface for enhance data.
+ *
+ * <p>
+ * Author: Wu Devin (haifwu@paypal.com)
+ * Date: 20/01/2018
```

Comment:
Could you use shifu default code template and add license header to source files?